task/01-strings-tasks.js

```diff
@@ -38,7 +39,8 @@ function concatenateStrings(value1, value2) {
  *   ''      => 0
  */
 function getStringLength(value) {
-    throw new Error('Not implemented');
+    const length = value.length
+    return length
```

Comment:
We can simply write `return value.length`, additional variable will be redundant in this case